soccerbot/launch/soccerbot.launch

```diff
@@ -11,14 +11,9 @@
         <arg name="y_pos" value="$(arg y_pos)" />
     </include>
 
-    <include file="$(find soccerbot)/launch/modules/localization.launch">
-        <arg name="robot_name" value="$(arg robot_name)"/>
-    </include>
+    <!--<include file="$(find soccerbot)/launch/modules/localization.launch">-->
```

Comment:
Still commetend this out. Please watch what you commit

Reply:
ok added back in